File: 3842-toggle-light-bulbs/3842-toggle-light-bulbs.go
func toggleLightBulbs(bulbs []int) []int {
    n := len(bulbs)
    freq := make([]bool, 101)

    for i := 0; i < n; i++ {
        if freq[bulbs[i]] {
            freq[bulbs[i]] = false
        } else {
            freq[bulbs[i]] = true
        }
    }
    ans := make([]int, 0)

    for i := 0; i < 101; i++ {
        if freq[i] {
            ans = append(ans, i)
        }
    }
    return ans
}

File: 3842-toggle-light-bulbs/3842-toggle-light-bulbs.py
class Solution:
    def toggleLightBulbs(self, bulbs: list[int]) -> list[int]:
        n = len(bulbs)
        freq = [False for _ in range(101)]


        for i in range(n):
            if freq[bulbs[i]]:
                freq[bulbs[i]] = False
            else:
                freq[bulbs[i]] = True
            
        ans = []
        for i in range(101):
            if freq[i]:
                ans.append(i)
        
        return ans